tests/integration/test_vulnerability_detector/test_feeds/test_download_feeds.py

```diff
@@ -1,5 +1,5 @@
 '''
-copyright: Copyright (C) 2015-2022, Wazuh Inc.
+copyright: Copyright (C) 2015-2023, Wazuh Inc.
```

Comment:
Apply the following change to all modified copyrights, please.
```suggestion
copyright: Copyright (C) 2015, Wazuh Inc.
```

Reply:
Done in commit: docs: Undo changes in wazuh_min_version and modify Copyright in some tests